src/CMakeLists.txt

```diff
@@ -34,6 +34,14 @@ if(NOT (BUILD_CLIENT OR BUILD_SERVER))
 endif()
 
 
+option(ENABLE_TOUCH "Enable Touchscreen support" FALSE)
+
+if(ENABLE_TOUCH)
```

Comment:
It'd be nice if this would verify the existence of `SEvent.TouchInput` here, not important though.

Reply:
I made the CMake script error if ``ENABLE_GLES`` is not enabled.